### PR TITLE
Github Actions to automate and monitor community issues

### PR DESCRIPTION
**Pull request by shub8968**
Special PR to automate community support processes as stipulated in the following issues:

* https://payara.atlassian.net/browse/SERV-943
* https://payara.atlassian.net/browse/SERV-944

### CODE REVIEW

**Comment by fturizo** [comment]
Currently reviewing these changes, please do not merge or delete yet.

File: .github/workflows/close-abandoned-issues.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Close Abandoned Issues
+
+on:
+  schedule:
+    - cron: "0 */24 * * *"
```
Comment (aubi) [inline .github/workflows/close-abandoned-issues.yaml:5]: Is it necessary to run it every hour? Isn't once a day enough?

Comment (fturizo) [reply .github/workflows/close-abandoned-issues.yaml:5]: As @shub8968 confirmed, it should run every day at midnight.

**Review by fturizo — APPROVED** [review]
LGTM

**Comment by fturizo** [comment]
jenkins test please

**Pull request merged**